anki-editor.el

```diff
@@ -790,14 +790,18 @@ and else from variable `anki-editor-prepend-heading'."
                                           note-type
                                           level
                                           prepend-heading))
-         (exported-fields (mapcar (lambda (x)
+         (exported-fields  (mapcar (lambda (x)
                                     (cons
                                      (car x)
                                      (anki-editor--export-string (cdr x)
                                                                  format)))
                                   fields)))
     (unless deck (user-error "Missing deck"))
     (unless note-type (user-error "Missing note type"))
+
+    ;; Sorting fields not necessary for Anki, but it removes
+    ;; randomness which breaks our tests.
+    (sort exported-fields (lambda (a b) (string< (car a) (car b))))
```

Comment:
@slotThe , I know that the speed is important for you. Let me know if you're not ok with this.

Reply:
The `exported-fields` list is tiny—I would be very surprised to have people with more than, say, five fields per card. This seems entirely justified to me if it makes testing easier.